content/get-started/cli-cmd-wallet-multisig-dealer-create.mdx

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
I would write somewhere in this page a warning like the one I suggested on #636

Reply:
Added!